lesson_l16/task_01/logging_config.ini

```diff
@@ -0,0 +1,34 @@
+[loggers]
```

Comment:
You can create once your logger_ini file for whole project in root. Then in all your modules you can unly reuse it.